clients/src/main/java/org/apache/kafka/clients/FetchSessionHandler.java

```diff
@@ -25,21 +25,21 @@
 import org.apache.kafka.common.requests.FetchRequest.PartitionData;
 import org.apache.kafka.common.requests.FetchResponse;
 import org.apache.kafka.common.utils.LogContext;
-import org.apache.kafka.common.utils.Utils;
 import org.slf4j.Logger;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Collections;
+import java.util.LinkedHashMap;
```

Comment:
Could you reduce the changes of imports? Also, the new order is weird to me as it is not a lexicographical order